### PR TITLE
Props tensor nnet

### PR DESCRIPTION
**Pull request by harlouci**
All __props__ checked in folder tensor/nnet/

### CODE REVIEW

File: theano/tensor/nnet/conv3d2d.py [hunk 1]
```diff
@@ -117,6 +113,8 @@ class IncDiagonalSubtensor(Op):
     """
     The gradient of DiagonalSubtensor
     """
+    __props__ = ("inplace")
```
Comment (lamblin) [inline theano/tensor/nnet/conv3d2d.py]: Here as well, it has to be a tuple: `("inplace",)`. Same thing at other places in that PR.

**Comment by nouiz** [comment]
travis fail.

**Comment by harlouci** [comment]
Not a clue...

2015-07-17 9:21 GMT-04:00 Frédéric Bastien <notifications@github.com>:

File: theano/tensor/nnet/neighbours.py [hunk 2]
```diff
@@ -13,6 +13,9 @@
 
 
 class Images2Neibs(Op):
+
+    __props__ = ("mode")
```
Comment (lamblin) [inline theano/tensor/nnet/neighbours.py]: Need a comma here as well.

**Pull request merged**